src/lib/serviceRoutines/postUpdateContext.cpp

```diff
@@ -82,6 +82,59 @@ std::string postUpdateContext
     return answer;
   }
 
+  //
+  // If upcr.contextElementResponseVector contains >= TWO responses and the second of them is a 'SccFound',
+  // then skip the first, the one that isn't 'SccFound'.
+  // Also, if the first of them is 472, then skip the second
+  //
+  // FIXME P6: Temporary hack for 'Entity found but Attribute not found':
+  //           See FIXME P6 'Temporary hack ...' 
+  //           in MongoCommonUpdate.cpp, function processContextElement
+  //
+
+
+  //
+  // Removing 'garbage' from the response vector (when just *one* incoming contextElement
```

Comment:
I think this comment would be more precise in the following way:

"Removing 'garbage' from the response vector (when then number of incoming contextElement *is different of the number of contextElement responses*)"